integrationtest/ValidationProject/ValidateStrykerResults.cs

```diff
@@ -107,10 +139,31 @@ public void SolutionRun()
 
             var report = JsonConvert.DeserializeObject<JsonReport>(strykerRunOutput);
 
+            CheckMutationKindsValidity(report);
```

Comment:
Make this method part of the CheckReportMutantCounts and rename it to CheckReportMutants